config/phase-0/connectors/mappings/mongodb-auth-code.json

```diff
@@ -0,0 +1,71 @@
+{
+	"target": "managed/Company",
+	"source": "system/CHSAuthCode/company_auth_codes",
+	"name": "systemChsauthcodeCompany_auth_codes_managedCompany",
+	"consentRequired": false,
+	"icon": null,
+	"displayName": "systemChsauthcodeCompany_auth_codes_managedCompany",
+	"properties": [{
+		"target": "authCode",
+		"source": "auth_code"
+	}, {
+		"target": "authCodeIsActive",
+		"source": "is_active",
+		"default": false
+	}, {
+		"target": "authCodeValidFrom",
+		"source": "valid_from"
+	}, {
+		"target": "number",
+		"source": "_id"
```

Comment:
does this map to the company number or the company ID? are they the same thing?

Reply:
Yes, they're the same thing. I think that it's required here since there's a correlation query that needed to be defined between the `_id` field in MongoDB and the `number` field in FIDC Company to match up the objects. It needs to be there because of that.